doc/classes/ProjectSettings.xml

```diff
@@ -1391,6 +1391,12 @@
 			Enabling this can greatly improve the responsiveness to input, specially in devices that need to run multiple physics frames per visible (process) frame, because they can't run at the target frame rate.
 			[b]Note:[/b] Currently implemented only on Android.
 		</member>
+		<member name="input_devices/buffering/android/use_accumulated_input" type="bool" setter="" getter="" default="true">
+			If [code]true[/code], multiple input events will be accumulated into a single input event when possible.
+		</member>
+		<member name="input_devices/buffering/android/use_input_buffering" type="bool" setter="" getter="" default="true">
+			If [code]true[/code], input events will be buffered prior to being dispatched.
+		</member>
```

Comment:
Shouldn't this imitate the previous entry "input_devices/buffering/agile_event_flushing" where it is a general parameter, but only implemented in Android?

Reply:
CC @RandomShaper who worked on this agile event flushing for Android.

Reply:
I'll merge for now as we need to get testing on these changes, and Android typically only gets tested when the changes are in a dev snapshot with export templates.

But the discussion/review can continue and we can improve the implementation later if relevant.